src/Controller/Admin/SystemCrudController.php

```diff
@@ -0,0 +1,105 @@
+        $sys_description = TextField::new('sys_description')->setLabel('entity.system.sys_description');
+        $sys_owner_subdepartment = TextField::new('sys_owner_subdepartment')->setLabel('entity.system.sys_owner_subdepartment');
+        $sys_emergency_setup = TextField::new('sys_emergency_setup')->setLabel('entity.system.sys_emergency_setup');
+        $sys_contractor = TextField::new('sys_contractor')->setLabel('entity.system.sys_contractor');
+        $sys_urgency_rating = TextField::new('sys_urgency_rating')->setLabel('entity.system.sys_urgency_rating');
+        $sys_number_of_users = TextField::new('sys_number_of_users')->setLabel('entity.system.sys_number_of_users');
+        $sys_technical_documentation = TextField::new('sys_technical_documentation')->setLabel('entity.system.sys_technical_documentation');
+        $sys_external_dependencies = TextField::new('sys_external_dependencies')->setLabel('entity.system.sys_external_dependencies');
+        $sys_important_information = TextField::new('sys_important_information')->setLabel('entity.system.sys_important_information');
+        $sys_superuser_organization = TextField::new('sys_superuser_organization')->setLabel('entity.system.sys_superuser_organization');
+        $sys_itsecurity_category = TextField::new('sys_itsecurity_category')->setLabel('entity.system.sys_itsecurity_category');
+        $sys_link_to_security_review = TextField::new('sys_link_to_security_review')->setLabel('entity.system.sys_link_to_security_review');
+        $sys_link_to_contract = TextField::new('sys_link_to_contract')->setLabel('entity.system.sys_link_to_contract');
+        $sys_end_of_contract = DateField::new('sys_end_of_contract')->setLabel('entity.system.sys_end_of_contract');
+        $sys_status = TextField::new('sys_status')->setLabel('entity.system.sys_status');
+        $sys_open_data = TextField::new('sys_open_data')->setLabel('entity.system.sys_open_data');
+        $sys_open_source = TextField::new('sys_open_source')->setLabel('entity.system.sys_open_source');
+        $sys_digital_post = TextField::new('sys_digital_post')->setLabel('entity.system.sys_system_category');
+        $sys_digital_transactions_pr_year = TextField::new('sys_digital_transactions_pr_year')->setLabel('entity.system.sys_digital_transactions_pr_year');
+        $sys_total_transactions_pr_year = TextField::new('sys_total_transactions_pr_year')->setLabel('entity.system.sys_total_transactions_pr_year');
+        //        $selfServiceAvailableFromItems = TextField::new('selfServiceAvailableFromItems')->setLabel('entity.system.selfServiceAvailableFromItems');
+        $sys_alternative_title = TextField::new('sys_alternative_title')->setLabel('sys_alternative_title');
+        $sys_version = TextField::new('sys_version')->setLabel('sys_version');
+
+        //      $coll_question = CollectionField::new('questions')->setEntryType('App\Form\CategoryType');
```

Comment:
Remember to remove unused code!
```suggestion
        $sys_alternative_title = TextField::new('sys_alternative_title')->setLabel('sys_alternative_title');
        $sys_version = TextField::new('sys_version')->setLabel('sys_version');
```